packages/oauth/src/stores/file-store.spec.js

```diff
@@ -0,0 +1,262 @@
+require('mocha');
+const { assert, expect } = require('chai');
+
+const sinon = require('sinon');
+const fs = require('fs');
+
+const { InstallProvider } = require('../index');
+const { FileInstallationStore } = require('./index');
+
+const clientSecret = 'MY_SECRET';
+const clientId = 'MY_ID';
+const stateSecret = 'stateSecret';
+
+const storedInstallation = {
+  team: {
+    id: 'test-team-id',
+    name: 'team-name',
+  },
+  enterprise: {
+    id: 'test-enterprise-id',
+    name: 'ent-name',
+  },
+  bot: {
+    token: 'botToken',
+    scopes: ['chat:write'],
+    id: 'botId',
+    userId: 'botUserId',
+  },
+  user: {
+    token: 'userToken',
+    id: 'userId',
+  },
+  incomingWebhook: {
+    url: 'example.com',
+    channel: 'someChannel',
+    channelId: 'someChannelID',
+    configurationUrl: 'someConfigURL',
+  },
+  appId: 'fakeAppId',
+  tokenType: 'tokenType',
+  isEnterpriseInstall: false,
+}
+
+// TODO: valid tests with org-wide installations
```

Comment:
We should add more tests for org-wide installations and token rotation use cases